In [24]:
#lista de 100 numeros
listaCreada = []
for iterable in range(80,201):
    listaCreada.append(iterable)
#print(listaCreada)

#uso de la lista previamente creada en el Ejercicio #1
#print(listaCreada)

from functools import reduce

promedioLista = (reduce( lambda a, b : (a + b)/ float(1+listaCreada.index((listaCreada[-1])))  , listaCreada) )
#

print("El promedio de La lista {} es: \n \nPromedio = {}".format(listaCreada,promedioLista))



El promedio de La lista [80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190, 191, 192, 193, 194, 195, 196, 197, 198, 199, 200] es: 
 
Promedio = 1.666597222222222


¡Nota! Esto no es un diagnóstico. Los cálculos que se proporcionan son estimaciones basadas en promedios.
Valor promedio	140
Suma de valor	16940
Cuenta promedio	121
Most Greater Value	200
Most Lowest Value	80
Multply by values	8.815580275870891e+257